contents/handbook/growth/marketing/ownership.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Marketing ownership
+sidebar: Handbook
+showTitle: true
+---
+
+### Owned by Content & Docs
+
+| &nbsp                           | **Individual owner** |
+|---------------------------------|----------------------|
+| **Alternative guides**          | TBD                  |
+| **AI integrations**             | Lior                 |
+| **Docs & tutorials**            | [See docs ownership](/handbook/content-and-docs/docs)   |
+| **Influencer sponsorships**     | Ian                  |
+| **Marketing dashboards**        | Andy                 |
+| **Newsletter**                  | Andy                 |
+| **Newsletter sponsorships**     | Ian                  |
+| **Newsletter sponsorships**     | Ian                  |
+| **SEO (general / technical)**   | Andy                 |
+| **SEO landing pages**           | Andy                 |
+| **Social media**                | Andy                 |
+| **Product comparisons**         | TBD                  |
+| **Where people hear about us**  | Andy                 |
+
+## Owned by Words & Pictures
+
```

Comment:
I think this is easier to find / more intuitive because these are mostly marketing functions. The alternative was killing the marketing section of the handbook altogether. As a newcomer, if I wanted to know who owned this stuff, I'd gravitate toward the marketing section.

Reply:
Sounds good